notifier/chatters.rst

```diff
@@ -332,6 +332,28 @@ a previous message to edit::
             ])
         );
 
+Answering Callback Queries in Telegram
+~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
```suggestion
Answering Callback Queries in Telegram
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
I don't quite understand why amount of `~` characters here should be more than in _Updating Telegram Messages_ title case above

Reply:
Oh, ok. Somehow I did not get it. Thank you, updated the file